app/serializers/invoice_item_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class InvoiceItemSerializer < ActiveModel::Serializer
+  attributes :id, :item_id, :invoice_id, :quantity, :unit_price, :created_at, :updated_at
```

Comment:
@case-eee  Yes, this is the data from the DB. We were under the impression that the models themselves would come through with all their data, but any model associations on other models would just come through with IDs. That seemed to be what the spec for the project was saying, at least as we interpreted it.